xml/decoders/rail_community/rcn277mapping.xml

```diff
@@ -0,0 +1,4222 @@
+<?xml version="1.0" encoding="utf-8"?>
+<?xml-stylesheet type="text/xsl" href="../XSLT/decoder.xsl"?>
```

Comment:
Change this line to reference xml/schema/decoder-4-15-2.xsd instead of xml/schema/decoder.xsd

That uses the most recent schema, which has additional support for `defaultItem`

The "Validate XML File.." selection in _PanelPro_ Debug menu will do the XML Schema checks on the updated file.

Reply:
More specifically
```
<variables xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://jmri.org/xml/schema/decoder-4-15-2.xsd">
```

Reply:
Thanks Bob, fixed 👍😁